generatePathMap.cjs.js

```diff
@@ -82,6 +82,9 @@ function generatePathMap(
     '/sdk/q/platform/flutter': {
       page: '/sdk/q/platform/[platform]'
     },
+    '/sdk/q/platform/react-native': {
```

Comment:
Adding this object here will force the next export to produce an index.html file for this route.  It is being included here because `/sdk` shows `react-native` as a platform choice but there is not generated `index.html` file for that route.  This will force that file to be generated even though it is just a choose your platform page which is the pattern that has been adopted for other similar situations.